controller/channel-billing-plan.go

```
@@ -0,0 +1,286 @@
+}
+
+func resolveMiniMaxRemainsURL(baseURL string) string {
+	if strings.Contains(baseURL, "minimax.io") {
+		return "https://api.minimax.io/v1/api/openplatform/coding_plan/remains"
+	}
+	return "https://api.minimaxi.com/v1/api/openplatform/coding_plan/remains"
+}
+
+// planUsageRemainingPercent returns the remaining percent of the tightest
+// window; this is what gets persisted for a plan channel.
+func planUsageRemainingPercent(usage *ChannelPlanUsage) float64 {
```

golangci-lint (2.12.2)

[error] func planUsageRemainingPercent is unused

(unused)